test/unit/helpers/config.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+
+var config = require('../../../helpers/config');
+
+var argvcloneStr = JSON.stringify(process.argv);
+
+describe('config is able to override AppConfig properties from the command line', function () {
+
+	const VERSION = '0.0.0a';
+
+	afterEach(() => {
+		process.argv = JSON.parse(argvcloneStr);
+	});
+
+	describe('config', function () {
+
+		it('single dash', function () {
+			process.argv.push('--config');
```

Comment:
is this the correct test? it looks like the same as the `double dash` test

Reply:
Nice catch. I have moved this test into the more generic test structure.